components/email/src/Email.svelte

```diff
@@ -781,11 +819,26 @@
                           <span class="name"
                             >{message.from[0].name ||
                               message.from[0].email}</span
-                          ><button class="email-tooltip-btn"
-                            ><DropdownSymbol /><span class="email-tooltip"
-                              >{message.from[0].email}</span
-                            ></button
                           >
+                          <button
+                            class="email-tooltip-btn"
+                            id={`button-${msgIndex}`}
+                            aria-describedby={message.id.slice(0, 3)}
+                            aria-expanded="false"
+                            aria-label="show email"
+                            on:click|stopPropagation={toggleTooltip}
+                          >
+                            <span class="icon-container" aria-hidden="true">
+                              <DropdownSymbol />
+                            </span>
+                            <span
+                              id={message.id.slice(0, 3)}
+                              role="tooltip"
+                              tabindex="0"
+                              class="email-tooltip"
+                              >{message.from[0].email}</span
```

Comment:
@heathergh not sure what is causing this, but if I click the chevron to show the tooltip - I can't seem to easily copy the email address (e.g. double click to highlight all). I'm testing this in chrome.

Reply:
Also cannot select all and copy in FF

Reply:
Copying & pasting the address wasn't part of the story AFAIK? https://app.clubhouse.io/nylas/story/65147/email-sender-receiver-s-email-should-appear-when-dropdown-is-clicked